serverless.yml

```diff
@@ -156,7 +159,7 @@ resources:
   pgDB:
     Type: "AWS::RDS::DBInstance"
     Properties:
-      DBName: "floods"
+      DBName: "postgres"
```

Comment:
Just curious as to why we're changing the DB name here

Reply:
Because if our default DBName is "floods" then floodsExists() will always be true, and we'll never seed on a new deployment. That's why it wasn't working before.

Reply:
Good catch!